health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/pages/employee/SetupMicroplan.js

```
@@ -409,7 +409,13 @@ const SetupMicroplan = ({ hierarchyType, hierarchyData }) => {
         <ActionBar
           style={{ zIndex: "19" }}
           setactionFieldsToRight
-          actionFields={[<Button label={t("GO_BACK_TO_MY_MICROPLAN")} onClick={() => history.goBack()} />]}
+          actionFields={[
+            <Button
+              label={t("GO_BACK_TO_MY_MICROPLAN")}
+              // onClick={() => history.goBack()}
+              onClick={() => history.push(`/${window.contextPath}/employee/microplan/microplan-search`)}
+            />,
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)